pgbench-wrapper/ci_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+source ci/common.sh
+
+# Build image for ci
+podman build --tag=quay.io/cloud-bulldozer/pgbench:snafu_ci -f pgbench-wrapper/Dockerfile . && podman push quay.io/cloud-bulldozer/pgbench:snafu_ci
+
+cd ripsaw
+
+sed -i 's/latest/snafu_ci/g' roles/pgbench/defaults/main.yml
+
+# Build new ripsaw image
+update_operator_image snafu_ci
+
+sed -i '/trap finish EXIT/d' tests/test_pgbench.sh
```

Comment:
could lines 17-29 get made into a common subroutine shared by all wrappers where you pass the ripsaw script to the subroutine?   This would shrink the CI code considerably.

Reply:
moved to get_uuid function. Good call!